src/region/RegionCreator.scss

```diff
@@ -8,5 +8,5 @@ $region_cursor_32_3x: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAY
 }
 
 .ba-RegionCreator-rect {
-    will-change: height, transform, width;
+    transform: translateZ(0);
```

Comment:
Why do we need this translate?

Reply:
It seems to help performance in Firefox and Safari a bit and has no apparent effect in Chrome. It's the same concept as using `translate3d` instead of `translate`. I can always remove it later if it becomes a problem.